pages/support.md

```diff
@@ -0,0 +1,75 @@
+<script>
+{
+	"title": "jQuery Support"
+}
+</script>
+
+## Community support
+
+The following learning resources are available online and free of charge:
+
+* [jQuery Learning Center](https://learn.jquery.com/)
+* [jQuery API Documentation](https://api.jquery.com/)
+* [jQuery UI Demos](https://jqueryui.com/demos/)
+* [jQuery UI API Documentation](https://api.jqueryui.com/)
+
+### Matrix chat
+
+jQuery is on Matrix! The support channel for jQuery is [#jquery_meeting:gitter.im](https://app.element.io/#/room/#jquery_meeting:gitter.im). You can [read the channel on Element](https://app.element.io/#/room/#jquery_meeting:gitter.im) without an account, or [join the channel](https://matrix.to/#/#jquery_meeting:gitter.im) via any Matrix client.
```

Comment:
`#jquery` is the support channel. `#jquery_meeting` is only for team members, but is publicly viewable.

Reply:
Oops, copypasta! Will fix.